spec/factories/dishes.rb

```diff
@@ -1,6 +1,7 @@
 FactoryGirl.define do
   factory :dish do
     name 'MyString'
+    category 'MyString'
```

Comment:
I'm surprised this is ok. Doesn't the factory need to have a "permitted" category?